src/main/java/ch/njol/skript/expressions/ExprHiddenPlayers.java

```diff
@@ -46,34 +29,40 @@ public class ExprHiddenPlayers extends SimpleExpression<Player> {
 				"[(all [[of] the]|the)] players hidden (from|for|by) %players%");
```

Comment:
Idk if `by` should be an option, sounds to represent something different from the other two

Reply:
removing this would break backwards compatibility, i havent altered the syntax, only the method name